src/Microsoft.ServiceFabric.Client.Http/ApplicationClient.cs

```diff
@@ -20,7 +20,7 @@ namespace Microsoft.ServiceFabric.Client.Http
     /// <summary>
     /// Class containing methods for performing ApplicationClient operataions.
     /// </summary>
-    internal partial class ApplicationClient : IApplicationClient
```

Comment:
Keep it as partial this interface is auto-generated from swagger specification, which generates all interfaces and classes as partial to allow manual extension later.

Reply:
Is the actual class also generated from swagger or just the interface? or is this just about having the same pattern for the interface and implementation?

Reply:
Both class and interface is autogenerated as partials. These are extended by adding partial class/interface in a file with samename but appended with an "_"